app-misc/tmux/tmux-3.1c.recipe

```diff
@@ -42,7 +49,7 @@ BUILD_PREREQUIRES="
 BUILD()
 {
 	autoreconf -fi
-	runConfigure ./configure
+	runConfigure ./configure --bindir=$commandBinDir
```

Comment:
should be:
```
runConfigure --omit-dirs binDir ./configure --bindir="$commandBinDir"
```

according to https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#using-the-commandbindir-user-variable

Reply:
Fixed. Thanks.